index.html

```diff
@@ -0,0 +1,99 @@
+                </p>
+                <p>
+                    It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.
+                </p>
+            </section>
+            <section class="main-section" id="Why_do_we_use_it">
+                <header class="h2">Why do we use it?</header>
+                <code>Why do we use it?</code>
+                <p>
+                    It is a long established fact that a reader will be distracted by the readable content of a page when looking at its layout. The point of using Lorem Ipsum is that it has standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book.
+                </p>
+                <ol>
+                    <li>At vero et odio </li>
+                    <li>eos et</li>
+                    <li>accusamus</li>
+                    <li>iusto</li>
+                    <li>dignissimos</li>
+                    <li>survived</li>
+                </ol>
+                <p>
+                    It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.
+                </p>
+            </section>
+            <section class="main-section" id="Where_can_I_get_some">
+                <header class="h2">Where can I get some?</header>
+                <code>Where can I get some?</code>
+                <p>
+                    There are many variations of passages of Lorem Ipsum available, but the majority have suffered alteration in some form, by injected humour standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book.
+                </p>
+                <p>
+                    It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.
+                </p>
+            </section>
+            <section class="main-section" id="standard_Lorem_Ipsum">
```

Comment:
You are using an unused CSS selector `main-section`

Reply:
I kindly suggest you read my comment above thanks! https://github.com/guillainbisimwa/Review-past-project/pull/2#discussion_r739682506 🔼